clients/src/test/java/org/apache/kafka/test/TestUtils.java

```diff
@@ -574,13 +576,15 @@ public static <T extends Throwable> void assertFutureThrows(
     public static void assertFutureError(Future<?> future, Class<? extends Throwable> exceptionClass)
         throws InterruptedException {
         try {
-            future.get();
+            future.get(5, TimeUnit.SECONDS);
```

Comment:
There is already a DEFAULT_MAX_WAIT_MS defined in this class, used to wait for conditions. Should we use the same for consistency?